builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -1243,3 +1342,15 @@ func (vm *virtualMachine) deployVirtualMachine(c *govmomi.Client) error {
 
 	return nil
 }
+
+// stringToBool is a helper function to convert a string to bool.
+func stringToBool(b string) (bool, error) {
```

Comment:
I am surprised that this does not exist in a helper lib. @phinze or @jen20 can you comment if this method should go somewhere under here: https://github.com/hashicorp/terraform/tree/master/helper